doc/releases/release-notes-3.3.rst

```diff
@@ -278,6 +294,8 @@ Bluetooth
 
   * Added local advertising address to bt_le_ext_adv_info.
 
+  * ISO: Change ISO seq_num to 16-bit
```

Comment:
This is another one, but I will change it later when I do a final sweep over the release notes.

Reply:
Fixed it myself :) But there's a similar one for mesh right below